Eatery Blue/UI/MenusScreen/MenusModelController.swift

```diff
@@ -29,7 +29,29 @@ class MenusModelController: MenusViewController {
     private lazy var loadCells: () = updateCellsFromState()
     
     private var selectedDay: Day = Day()
-    private var currentMealType: String = "Breakfast"
+    private var currentMealType: String = {
+        let date = Date()
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "HH"
+        if let hour : Int = Int(dateFormatter.string(from: date)) {
+            dateFormatter.dateFormat = "mm"
+            if let minute : Int = Int(dateFormatter.string(from: date)){
+                if hour < 10 || hour == 10 && minute < 30 || hour >= 23 || hour == 22 && minute >= 30 {
+                    return "Breakfast"
+                } else if hour == 10 && minute >= 30 || 10 < hour && hour < 14 || hour == 14 && minute < 30 {
+                    return "Lunch"
+                } else if hour == 14 && minute >= 30 || 14 < hour && hour < 20 || hour == 20 && minute < 30 {
+                    return "Dinner"
+                } else {
+                    return "Late Dinner"
+                }
+            } else {
+                return "Breakfast"
+            }
+        } else {
+            return "Breakfast"
+        }
```

Comment:
This should be filtered out to a `String+Extension` file with proper docstring comments for the function.